heksher/api/v1/rules.py

```diff
@@ -120,6 +120,7 @@ class QueryRulesOutput_Rule(ORJSONModel):
     context_features: List[Tuple[str, str]] = Field(
         description="a list of exact-match conditions for the rule, in hierarchical order"
     )
+    rule_id: int
```

Comment:
```suggestion
    rule_id: int = Field(description=...)
```

Reply:
The rules are managed using rule IDs, I think it's elementary information.